django/test/client.py

```diff
@@ -592,39 +592,48 @@ def login(self, **credentials):
         are incorrect, or the user is inactive, or if the sessions framework is
         not available.
         """
-        from django.contrib.auth import authenticate, login
+        from django.contrib.auth import authenticate
         user = authenticate(**credentials)
         if (user and user.is_active and
                 apps.is_installed('django.contrib.sessions')):
-            engine = import_module(settings.SESSION_ENGINE)
+            self._login(user)
+            return True
+        else:
+            return False
 
-            # Create a fake request to store login details.
-            request = HttpRequest()
+    def force_login(self, user, backend=None):
```

Comment:
Agreed, but from a user's perspective I don't see the difference between `login()` and `login_user()`. I could even think that it would prevent staff/superusers to login.